apps/kitty-krew/src/app/routeTree.gen.ts

```
@@ -21,6 +22,12 @@ const IndexRoute = IndexImport.update({
   getParentRoute: () => rootRoute,
 } as any)
 
+const PodsNamespacePodNameRoute = PodsNamespacePodNameImport.update({
+  id: '/pods/$namespace/$podName',
+  path: '/pods/$namespace/$podName',
+  getParentRoute: () => rootRoute,
+} as any)
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)